ui/src/layouts/storage/mod.rs

```diff
@@ -75,49 +74,29 @@ pub enum ChanCmd {
         new_name: String,
     },
     DeleteItems(Item),
-    SendFileToChat {
-        files_path: Vec<PathBuf>,
-        conversation_id: Uuid,
-    },
 }
 
-#[derive(PartialEq, Props)]
-pub struct Props {
-    send_files_to_chat_mode: Option<UseState<bool>>,
-    chat_id: Option<Uuid>,
+#[derive(Props)]
+pub struct Props<'a> {
+    storage_files_to_chat_mode_is_active: Option<UseState<bool>>,
```

Comment:
I don't like the name `storage_files_to_chat_mode_is_active`. It looks like this variable is used to denote a file picker mode. Something like `is_selecting_files` would be better. 

I don't like the name `on_files_selected_to_send`. Prefer `on_files_attached`. 

After looking at the rest of the file, it looks like a separate Layout is warranted. The FilesLayout is different from a file picker. Combining those two into one view causes clutter and coupling. This seems especially unnecessary in the storage view because the refactor was supposed to separate the control logic from the view.